src/Pelagos/Bundle/AppBundle/Resources/views/MaintenanceMode/maintenanceMode.html.twig

```diff
@@ -0,0 +1,40 @@
+{% extends "PelagosAppBundle:template:UI.html.twig" %}
+{% block title %}Maintenance Mode{% endblock %}
+
+{% block javascripts %}
+{{ parent() }}
+
+<script type="text/javascript">
+$(document).ready(function()
+{
+    $("[name=bannermode]").change(function () {
+        console.log($(this).val());
```

Comment:
Debug code

Reply:
at least it's not anything end-user facing